Jamfile

```diff
@@ -276,10 +273,12 @@ rule building ( properties * )
 
 	if <toolset>msvc in $(properties) || <toolset>intel-win in $(properties)
 	{
-		# allow larger .obj files (with more sections)
-		result += <cflags>/bigobj ;
 		# https://docs.microsoft.com/en-us/cpp/build/reference/utf-8-set-source-and-executable-character-sets-to-utf-8?view=msvc-170
 		result += <cflags>/utf-8 ;
+		# https://devblogs.microsoft.com/cppblog/msvc-now-correctly-reports-__cplusplus/
+		result += <cxxflags>/Zc:__cplusplus ;
```

Comment:
I don't know why it is unrecognized:
https://ci.appveyor.com/project/arvidn/libtorrent/builds/42059904/job/e9gkxsmfgwujpxy0#L125
Any ideas?

Reply:
It is because of `:` and it need double quotes around it.

Reply:
unfortunate that appveyor CI still don't recognize it:
https://ci.appveyor.com/project/arvidn/libtorrent/builds/42060994/job/owm1w4ublhnvlyw0#L124
while Github Actions have no problem:
https://github.com/arvidn/libtorrent/runs/4685279536?check_suite_focus=true#step:5:36

Reply:
I think I found the root cause. 
Appveyor is using msvc `14.0` which translate to "Visual Studio 2015 version 14.0" but `/Zc:__cplusplus` only has support starting in "Visual Studio 2017 version 15.7".
Now libtorrent RC_1_2 supports C++11 which also includes "Visual Studio 2015" and that means `/Zc:__cplusplus` isn't available. So none of the build system should have `/Zc:__cplusplus`. I'll make the change soon.

Reply:
On the other hand, code is already using `__cplusplus` and removing the flag may cause trouble. Forget it, I'll leave this absurd state as is...